umap/static/umap/js/modules/sync/updaters.js

```diff
@@ -89,9 +89,10 @@ export class FeatureUpdater extends BaseUpdater {
       console.error(`Unable to find feature with id = ${metadata.id}.`)
     }
     switch (key) {
-      case 'geometry':
+      case 'geometry': {
```

Comment:
Don't get this one

Reply:
The error is that one: https://biomejs.dev/linter/rules/no-switch-declarations/ maybe we should use a `if` for that one-case switch?

Reply:
Something like that:

```suggestion
    if (key === 'geometry') {
      const datalayer = this.getDataLayerFromID(metadata.layerId)
      this.getDataLayerFromID(metadata.layerId).geoJSONToLeaflet({
        geometry: value,
        id: metadata.id,
        feature,
      })
    } else {
      this.updateObjectValue(feature, key, value)
      feature.datalayer.indexProperties(feature)
    }

```